packages/design-system/src/components/FormLabel/__snapshots__/FormLabel.test.jsx.snap

```diff
@@ -186,6 +186,7 @@ exports[`FormLabel renders requirementLabel string 1`] = `
 exports[`FormLabel supports additional classNames 1`] = `
```

Comment:
could we update this test case to be "supports additional attributes", and pass in something other than `labelClassName`?